skiko/src/jsMain/kotlin/org/jetbrains/skiko/CanvasRenderer.kt

```diff
@@ -32,15 +31,13 @@ abstract class CanvasRenderer constructor(htmlCanvas: HTMLCanvasElement, val wid
     val canvas: Canvas
         get() = surface.canvas
 
-    abstract fun drawFrame(currentTimestamp: Double)
+    abstract fun drawFrame()
 
     fun needRedraw() {
-        window.requestAnimationFrame { timestamp ->
-            GL.makeContextCurrent(contextPointer)
-            canvas.clear(-1)
-            drawFrame(timestamp)
-            surface.flushAndSubmit()
-            context.flush()
-        }
+        GL.makeContextCurrent(contextPointer)
```

Comment:
`needRedraw` by itself shouldn't do any drawing operations (`canvas.clear(-1)`, `context.flush()`, etc).

It should only inform drawing loop to schedule drawing. `requestAnimationFrame` exists for this purpose, but calling it multiple times will schedule too many draw requests.

Reply:
I'll open a new PR with this change